### PR TITLE
Test certificates age, so we are sure the certificates were just generated during assemble script

### PR DESCRIPTION
**Pull request by hhorak**
**WARNING**: This PR requires https://github.com/sclorg/container-common-scripts/pull/245 to be merged first.

### CODE REVIEW

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
[test]

**Comment by phracek** [comment]
let's tests RHEL9 - unsubscribed host
[test-all]

**Comment by phracek** [comment]
[test-all]

**Review by phracek — APPROVED** [review]
The test suite is present and working properly. I did not hit any issues.

File: 2.4/test/run [hunk 1]
```diff
@@ -141,6 +141,23 @@ function run_s2i_test() {
...
   run "ct_test_response '${cip}:8080' 200 'This is a sample s2i application with static content.'"
+
+  # Let's see whether the automatically generated certificate works as expected
+  run "curl -k https://${cip}:8443 >output_generated_ssl_cert"
+  run "fgrep -e 'This is a sample s2i application with static content.' output_generated_ssl_cert"
+
+  # We also need to make sure the certificate is generated no sooner than in assemble phase,
+  # because shipping the same certs in the image would make it easy to exploit
+  # Let's see how old the certificate is and compare with how old the image is
+  image_age_s=$(ct_get_image_age_s "${IMAGE_NAME}")
```
Comment (zmiklank) [inline 2.4/test/run:152]: Wouldn't it make sense here to create a separate test case for checking certificate age?

Comment (hhorak) [reply 2.4/test/run]: The reason to put it here was to not need to build a new application image again, which would be the same as the default one. I didn't want to introduce any assumptions that e.g. one test would depend on an application image built from previous test being available. On the other hand, building a new image should not take too much time, so we can likely separate it into a new test.

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
[test]

**Comment by zmiklank** [comment]
The https://github.com/sclorg/container-common-scripts/pull/245 is now merged. Could you please rebase common to latest master? I can do it too, if you'll be OK with that.

**Comment by phracek** [comment]
[test-all]

**Review by jamacku — APPROVED** [review]

**Pull request merged**